src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -502,6 +504,7 @@ def run(self, args, unknown_args):
         self.collect_commcare_cloud_details()
         self.collect_commcare_hq_details()
         self.validate_environment_settings()
+        self.collect_control_machine_os_level_info()
```

Comment:
nit: I reckon all of these could be private functions?

Reply:
True. I'll update these to private